.github/workflows/codeql.yml

```diff
@@ -0,0 +1,56 @@
+name: 'CodeQL'
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ['master']
+  schedule:
+    - cron: '30 12 * * 5'
+
+permissions:
+  actions: read
+  contents: read
+  security-events: write
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ['javascript']
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Initialize CodeQL
+        uses: signavio/codeql-action/init@v2
+        with:
+          queries: security-extended
+          # run an 'alert-suppression' query
+          packs: 'codeql/${{ matrix.language }}-queries:AlertSuppression.ql'
+          languages: ${{ matrix.language }}
+
+      - name: auto-build
+        uses: signavio/codeql-action/autobuild@v2
```

Comment:
I assume the `autobuild` is not part of the `signavio/codeql-action` you would need to use another parent here.